src/app/ui/color_sliders.cpp

```diff
@@ -373,7 +373,7 @@ void ColorSliders::addSlider(const Channel channel,
   item.box       = new HBox();
   item.absSlider = new Slider(absMin, absMax, 0);
   item.relSlider = new Slider(relMin, relMax, 0);
-  item.entry     = new ColorEntry(item.absSlider, item.relSlider);
+  item.entry     = new ExprEntry();
```

Comment:
With this change we lost the behavior of `ColorEntry` (we cannot use up/down arrow keys to modify the value).